includes/db_api.php

```diff
@@ -0,0 +1,218 @@
+<?php
+
+include_once 'config.php';
+
+function registerUser($name, $username, $password, $user_code, $date_creation, $points, $isAdmin = false) {
```

Comment:
Sugestão: ao nível da BD e em APIs à volta da BD tenta usar termos como `CREATE`, `READ`, `UPDATE` e `DELETE` (`CRUD`). É linguagem mais próxima do domínio e deixa mais explicito o que acontece com aquela operação.

Em cima disto, podes ter um outro objecto, `repository`, `authController`, whatever, que trata de toda a lógica de validar password, verificar se o user já existe, etc e deixar este ficheiro para operações mais unitárias.

```suggestion
function createUser($name, $username, $password, $user_code, $date_creation, $points, $isAdmin = false) {
```